lib/convection/control/cloud.rb

```diff
@@ -16,17 +16,21 @@ def stacks
         @cloudfile.stacks
       end
 
-      def deck
-        @cloudfile.deck
+      def stack_groups
+        @cloudfile.stack_groups
       end
 
-      def converge(to_stack, &block)
-        unless to_stack.nil? || stacks.include?(to_stack)
-          block.call(Model::Event.new(:error, "Stack #{ to_stack } is not defined", :error)) if block
-          return
-        end
+      def filter_deck(options = {}, &block)
+        included_stacks = stack_groups[options[:stack_group]]
+        included_stacks ||= options.fetch(:stack_list, [])
```

Comment:
Since the default value isn't returned here in any case we can just use `Hash#[]` instead of trying to use fetch's default option.